pages/common/tslint.md

```diff
@@ -0,0 +1,19 @@
+# tslint
+
+> A pluggable linting utility for TypeScript.
+
+- Create tslint config:
+
+`tslint --init`
+
+- Lint on files:
+
+`tslint {{filename}}.js {{filename1}}.js`
+
+- Fix lint issues:
+
+`tslint --fix`
+
+- Lint with config in project root:
```

Comment:
Is this a config file in a specific directory, or a specific config file?

Reply:
The config `tslint.json` needs to be in the project root path. Similiar to others `--config {{path/to/config}}` but there you need to specify the root path where the config is located.

Reply:
Ah, I see. Perhaps this will read better then?

```suggestion
- Lint with the config file in the project root:
```